core/src/mindustry/world/blocks/distribution/AutotilerBuilding.java

```diff
@@ -0,0 +1,9 @@
+package mindustry.world.blocks.distribution;
+
+import mindustry.gen.Building;
+import mindustry.world.Block;
+
+public interface AutotilerBuilding{
```

Comment:
Why is this called `AutotilerBuilding` when it has nothing to do with tiling itself...?

Reply:
Because I check if block is Autotiler in InputHandler. Both conduits and conveyors are inherited from it.